### ISSUE TITLE
Another release depending on kryo-shaded 4.0.2

### ISSUE THREAD

**Issue by jzhuge** [magro/kryo-serializers#120]
Is it possible to push another release based on 0.43 with dependency `kryo-shaded` instead of `kryo`?

We are not ready for kryo v5 yet and prefer `kryo-shaded` to `kryo` in order to avoid conflicts of `org.ow2.asm:asm`.

**Comment by magro** [comment]
Could you exclude the dependency on kryo when depending on kryo-serializers? And depend on kryo-shaded explicitly?

**Comment by jzhuge** [comment]
Yes it works but it is the kind of thing (as well as custom Gradle dependency substitution strategy) I am trying to avoid.

**Comment by magro** [comment]
Sure. Let me see when I find the time for this.
